### PR TITLE
[To NewDoc] Update the typos of UserMannual

### PR DESCRIPTION
**Pull request by liuzhen1207**
e.g
Wrong character
wrong version info
Inconsistent result sets

### CODE REVIEW

**Review by wangchao316 — APPROVED** [review]
Good, Thanks for contribution
LGTM

File: docs/zh/UserGuide/Concept/Encoding.md [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 ## 编码方式
 
-为了提高数据的存储效率，需要在数据写入的过程中对数据进行编码，从而减少磁盘空间的使用量。在写数据以及读数据的过程中都能够减少I/O操作的数据量从而提高性能。IoTDB支持四种针对不同类型的数据的编码方法：
+为了提高数据的存储效率，需要在数据写入的过程中对数据进行编码，从而减少磁盘空间的使用量。在写数据以及读数据的过程中都能够减少I/O操作的数据量从而提高性能。IoTDB支持五种针对不同类型的数据的编码方法：
```
Comment (HTHou) [inline docs/zh/UserGuide/Concept/Encoding.md:24]:
```suggestion
为了提高数据的存储效率，需要在数据写入的过程中对数据进行编码，从而减少磁盘空间的使用量。在写数据以及读数据的过程中都能够减少I/O操作的数据量从而提高性能。IoTDB支持多种针对不同类型的数据的编码方法：
```

File: docs/zh/UserGuide/Operation Manual/DML Data Manipulation Language.md [hunk 2]
```diff
@@ -1073,7 +1058,7 @@ select * from root.ln.wf01.wt01 where time > 2017-11-01T00:05:00.000 and time <
 
 意思是：
 
-所选设备为ln组wf01工厂wt01设备； 所选时间序列是该设备下的第一列，即电源状态。 SQL语句要求在“ 2017-11-01T00：05：00.000”和“ 2017-11-01T00：12：00.000”的时间点之间选择状态传感器值。
+所选设备为ln组wf01工厂wt01设备； 所选时间序列是该设备下的第二列，即温度。 SQL语句要求在“ 2017-11-01T00：05：00.000”和“ 2017-11-01T00：12：00.000”的时间点之间选择温度传感器值。
```
Comment (HTHou) [inline docs/zh/UserGuide/Operation Manual/DML Data Manipulation Language.md:1061]:
```suggestion
所选设备为ln组wf01工厂wt01设备； 所选时间序列是该设备下的第二列，即温度。 SQL语句要求在"2017-11-01T00:05:00.000"和"2017-11-01T00:12:00.000"的时间点之间选择温度传感器值。
```

File: docs/zh/UserGuide/Operation Manual/DML Data Manipulation Language.md [hunk 3]
```diff
@@ -1102,7 +1087,7 @@ select * from root.ln.wf01.wt01 where time > 2017-11-01T00:05:00.000 and time <
 
 意思是：
 
-所选设备为ln组wf01工厂wt01设备； 所选时间序列是该设备下的第二列，即温度。 SQL语句要求在“ 2017-11-01T00：05：00.000”和“ 2017-11-01T00：12：00.000”的时间点之间选择温度传感器值。
+所选设备为ln组wf01工厂wt01设备； 所选时间序列是该设备下的第一列，即电源状态。 SQL语句要求在“ 2017-11-01T00：05：00.000”和“ 2017-11-01T00：12：00.000”的时间点之间选择状态传感器值。
```
Comment (HTHou) [inline docs/zh/UserGuide/Operation Manual/DML Data Manipulation Language.md:1090]:
```suggestion
所选设备为ln组wf01工厂wt01设备； 所选时间序列是该设备下的第一列，即电源状态。 SQL语句要求在" 2017-11-01T00:05:00.000"和"2017-11-01T00:12:00.000"的时间点之间选择状态传感器值。
```

File: docs/zh/UserGuide/Operation Manual/DML Data Manipulation Language.md [hunk 4]
```diff
@@ -1261,6 +1242,7 @@ select s1 as temperature, s2 as speed from root.ln.wf01.wt01;
 ```   sql
 select s1, sin(s1), cos(s1), tan(s1) from root.sg1.d1 limit 5 offset 1000;
 ```
+select s1,s2 from root.ln.* ALIGN BY DEVICE
```
Comment (HTHou) [inline docs/zh/UserGuide/Operation Manual/DML Data Manipulation Language.md:1245]:
```suggestion
```

File: docs/zh/UserGuide/Operation Manual/DML Data Manipulation Language.md [hunk 5]
```diff
@@ -1186,7 +1167,7 @@ select * from root.ln.wf01.wt01 limit 10 offset 100 slimit 2 soffset 0
 
 意思是：
 
-所选设备为ln组wf01工厂wt01设备； 所选时间序列是此设备下的第0列至第1列（第一列编号为第0列）。 SQL语句子句要求返回查询结果的第100至109行（第一行编号为0行）。
+所选设备为ln组wf01工厂wt01设备； 所选时间序列是此设备下的第0列至第1列（第一列编号为第0列）。 SQL语句子句要求返回查询结果的第0至9行（第一行编号为0行）。
```
Comment (HTHou) [inline docs/zh/UserGuide/Operation Manual/DML Data Manipulation Language.md:1170]:
```suggestion
所选设备为ln组wf01工厂wt01设备； 所选时间序列是此设备下的第0列至第1列（第一列编号为第0列）。 SQL语句子句要求返回查询结果的第100至109行（第一行编号为0行）。
```

**Pull request merged**